### PR TITLE
[CORDA-1612]: Fix truncated code snippets in docs PDF

### PR DESCRIPTION
**Pull request by sollecitom**
https://r3-cev.atlassian.net/browse/CORDA-1612

### CODE REVIEW

File: docs/source/changelog.rst [hunk 1]
```diff
@@ -7,6 +7,8 @@ release, see :doc:`upgrade-notes`.
 Unreleased
 ==========
 
+* Improved docs PDF quality. Building the docs now requires ``LaTex`` to be installed on the OS.
```
Comment (tlil) [inline docs/source/changelog.rst]: docs => documentation

File: docs/source/key-concepts-flows.rst [hunk 2]
```diff
@@ -22,7 +22,7 @@ what order.
 
 Here is a visualisation of the process of agreeing a simple ledger update between Alice and Bob:
 
-.. image:: resources/flow.gif
```
Comment (tlil) [inline docs/source/key-concepts-flows.rst]: We should rethink this - presumably the GIF is telling a story on the HTML doc site. Obviously needs to be static in the PDF, but in HTML it's telling a story at the moment. @joeldudleyr3 any thoughts on what we do with these two GIFs moving forward? Having them on the HTML site is fine IMHO, but doesn't translate well into a static, printable doc.

File: docs/source/index.rst [hunk 3]
```diff
@@ -52,7 +50,6 @@ We look forward to seeing what you can do with Corda!
    azure-vm.rst
    aws-vm.rst
    loadtesting.rst
-
 .. toctree::
    :caption: Design docs
```
Comment (tlil) [inline docs/source/index.rst]: This disappears with the file renames further up; however, we _do_ want to have the design docs on the HTML site. Could we use a different TOC for HTML and PDF? Or use a topic filter for PDF?

Comment (sollecitom) [reply docs/source/index.rst]: Fixed

File: docs/make-docsite.sh [hunk 4]
```diff
@@ -13,11 +13,10 @@ fi
 
 # TODO: The PDF rendering is pretty ugly and can be improved a lot.
```
Comment (tlil) [inline docs/make-docsite.sh]: Is this comment still true? :) It's better now, no? ;)

File: docs/source/key-concepts-flows.rst (hunk 2 above)
Comment (sollecitom) [reply docs/source/key-concepts-flows.rst]: It does.

Comment (tlil) [reply docs/source/key-concepts-flows.rst]: Yes, this comment was from before you made the change. I've approved now ;P

**Pull request merged**